combinations/hash.tsv

```diff
@@ -517,3 +517,4 @@ gatk4=4.4.0.0,samtools=1.18,coreutils=9.3
 r-base,r-dplyr,r-optparse,r-devtools,r-topicmodels,r-philentropy,r-metrics,r-jsonlite
 presto=0.7.1,igblast=1.21.0,wget=1.20.1,biopython=1.79	quay.io/bioconda/base-glibc-busybox-bash:2.1.0	0
 bioconductor-limma=3.58.1,bioconductor-edger=4.0.2,r-data.table=1.14.10,r-gplots=3.1.3,r-statmod=1.5.0
+r-nacho=2.0.6,r-tidyverse=2.0.0,r-ggplot2=3.4.4,r-rlang=1.1.3,r-tidylog=1.0.2,r-fs=1.6.2,bioconductor-complexheatmap=2.18.0,r-circlize=0.4.15,r-yaml=2.3.8,r-ragg=1.2.7,r-rcolorbrewer=1.1_3,r-pheatmap=1.0.12,pandoc=3.1.3,gmp=6.3.0
```

Comment:
Can we decompose this? Looks like a lot of different functionality...

Reply:
Yeah not sure if we fully can. The scripts do both running of the functionality as well as some plotting

Reply:
@bgruening  - I made 3 distinct containers out of this and also did not import tidyverse entirely but rather only the packages absolutely necessary. Should be better that way :)